components/o-toggle/main.js

```diff
@@ -5,6 +5,6 @@ const constructAll = () => {
 	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
-document.addEventListener('o.DOMContentLoaded', constructAll);
+if (document) document.addEventListener('o.DOMContentLoaded', constructAll);
```

Comment:
this'll still error if document is not defined.

```suggestion
if (typeof document !== 'undefined') document.addEventListener('o.DOMContentLoaded', constructAll);
```

this should work though, right?